agent/security_helper.py

```diff
@@ -336,3 +340,32 @@ def cpu_vulnerabilities():
         res['mitigations_disabled'] = mitigations_disabled
 
     return res
+
+
+def patch_sshd_config(patch_param, safe_value):
+    if not os.path.isfile(SSHD_CONFIG_PATH):
+        return
+    with open(SSHD_CONFIG_PATH, 'r+') as sshd_config:
+        replaced = False
+        lines = sshd_config.readlines()
+        patched_lines = []
+        for line in lines:
+            patched_lines.append(line)
+            line = line.strip()
+            if not line or line[0] == '#':
+                # skip empty lines and comments
+                continue
+            line_split = line.split(maxsplit=1)
+            if len(line_split) != 2:
+                continue
+            param, value = line_split
+            value = value.strip('"')
+            if param == patch_param and value != safe_value:
+                logger.info('{}: replacing "{}" with "{}"'.format(param, value, safe_value))
+                patched_lines[-1] = param + ' ' + safe_value + '\n'
```

Comment:
Can we annotate the end of this line with something like `# Added by wott-agent on $TIMESTAMP`?

Reply:
We can only add a comment above or below the line, not to the line itself.

Reply:
This is not true:

```
$ grep Proto /etc/ssh/sshd_config
Protocol 2 #This is a comment
$ sudo service ssh restart
$ sudo /usr/sbin/sshd -t
$ echo $?
0
```

Reply:
That said, I agree that praxis is to add it above, so maybe we should stick to that.